### ISSUE TITLE
module 'matplotlib.cbook' has no attribute 'is_numlike'

### ISSUE THREAD

**Issue by Anguslam228** [networkx/networkx#4764]
Hi all,
I'm fairly new to Networkx, so I'm not sure if the error is common, and I can't make much sense of the error documentation. 

### Current Behavior

Plot is missing edges, as shown below:
![networkx_issue](https://user-images.githubusercontent.com/47729178/116293764-136a0380-a74c-11eb-91ea-7491207db567.png)

### Expected Behavior

![image](https://user-images.githubusercontent.com/47729178/116293805-195fe480-a74c-11eb-94b3-56e0ba6ac951.png)

### Steps to Reproduce

Any time I try to run nx.draw_networkx(g), the error occurs

### Environment

Python version: 3.8
NetworkX version: 2.3

**Comment by dschult** [comment]
It looks like your version of matplotlib doesn't match your version of networkx.  You can try to use a more recent networkx version, or an older version of matplotlib. I would recommend upgrading to a newer networkx:  `pip install networkx==2.5`

**Comment by Anguslam228** [comment]
I just upgraded networkx, but the issue still persists. Anything else I could try?

**Comment by Anguslam228** [comment]
Update: I downgraded matplotlib and it works now. Is this a viable solution in the long run?

**Comment by dschult** [comment]
I don't think you successfully upgraded networkx. This was fixed in #3179 back in 2018.
Perhaps you upgraded NX on one version of python but are running it on another?

Downgrading matplotlib will also work, but is not a long term solution (or in my opinion not a good short term solution).

**Comment by rossbar** [comment]
I agree with @dschult , I'm going to close this one. If issues persist, please feel free to reopen along with version info from the session where you're experiencing the problem (e.g. `import networkx; networkx.__version__`)
